samples/test/authViewTutorial.test.js

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const {assert} = require('chai');
+const {describe, it, before, after} = require('mocha');
+const cp = require('child_process');
+const uuid = require('uuid');
+
+const {BigQuery} = require('@google-cloud/bigquery');
+
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+const generateUuid = () => `gcloud-tests-${uuid.v4()}`.replace(/-/gi, '_');
```

Comment:
maybe give these a more descriptive uuid prefix so you can find what's creating them if things go badly and they don't get cleaned up?   gcloud-tests seems unnecessarily vague (and would have me seeking out something in the gcloud sdk if I saw them).